Sources/Widgets/Representations/ArrowHandleRepresentation/index.js

```diff
@@ -176,6 +245,9 @@ function vtkArrowHandleRepresentation(publicAPI, model) {
 
 const DEFAULT_VALUES = {
   defaultScale: 1,
+  orientation: [0, 0, 0],
+  viewMatrix: mat4.create(),
```

Comment:
You should not have that `mat4.create()` inside the defaultValues. You should instead put something like below in the extend function.

```
model.viewMatrix = new Float64Array(16);
```

And just leave that line in defaults but commented out.

Reply:
why is it not recommanded to do so?

Reply:
because the same typed array will be shared across all widget instances